### PR TITLE
Fix unattended_install issue

### PR DESCRIPTION
**Pull request by qingwangrh**
1. Support ovmf install
2. Reduce disk number for windows virtio-blk installation
ID:1714485
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads,unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2016 --driveformat=virtio_blk,virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35,i440fx --customsparams="vm_mem_limit = 12G\nimage_aio=threads\ncd_format=ide" --clone=no --firmware=ovmf

(1/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (623.35 s)
 (2/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (636.89 s)
 (3/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (630.35 s)
 (4/4) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (638.61 s)

python ConfigTest.py --testcase=unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads,unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2012 --driveformat=virtio_blk,virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35,i440fx --customsparams="vm_mem_limit = 12G\nimage_aio=threads\ncd_format=ide" --clone=no

(01/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (741.01 s)
 (02/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (727.64 s)
 (03/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (713.05 s)
 (04/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (724.95 s)
 (05/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (760.06 s)
 (06/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (717.60 s)
 (07/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (844.03 s)
 (08/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (1457.55 s)
 (09/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (934.09 s)
 (10/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (833.88 s)
 (11/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (744.93 s)
 (12/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (714.88 s)
 (13/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (755.65 s)
 (14/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (719.20 s)
 (15/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (758.11 s)
 (16/16) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (716.23 s)

**Comment by qingwangrh** [comment]
@YongxueHong @menli820 @peixiu Please help to review,thanks.

File: generic/tests/cfg/unattended_install.cfg [hunk 1]
```diff
@@ -43,6 +43,12 @@
     # This value can be set to 'yes' if required to perform installation trigger
     # random time in the range random.randint(0, install_trigger_time)
     random_trigger = "no"
+
```
Comment (menli820) [inline generic/tests/cfg/unattended_install.cfg:46]: suggestion remove this blank line

Comment (qingwangrh) [reply generic/tests/cfg/unattended_install.cfg]: ok

**Comment by menli820** [comment]
LGTM

python3 ConfigTest.py --testcase=unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads,unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2019 --driveformat=virtio_blk --nicmodel=virtio_net --imageformat=qcow2 --machines=q35,i440fx --customsparams="vm_mem_limit = 12G\nimage_aio=threads\ncd_format=ide" --clone=no --firmware=ovmf

 (1/2) Host_RHEL.m8.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (991.01 s)
 (2/2) Host_RHEL.m8.u5.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: /^[[PASS (1019.58 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

File: generic/tests/cfg/unattended_install.cfg [hunk 2]
```diff
@@ -43,6 +43,11 @@
     # This value can be set to 'yes' if required to perform installation trigger
     # random time in the range random.randint(0, install_trigger_time)
     random_trigger = "no"
+    ovmf:
```
Comment (PaulYuuu) [inline generic/tests/cfg/unattended_install.cfg:46]: Already included in VT.
https://github.com/avocado-framework/avocado-vt/blob/6681e9245af2f2c130775c1a0d7eab134216c553/virttest/shared/cfg/guest-hw.cfg#L460-L470

Comment (qingwangrh) [reply generic/tests/cfg/unattended_install.cfg:46]: remove same.

**Comment by CongLi** [comment]
@PaulYuuu Yihuang, please review the PR again, thanks.

**Comment by peixiu** [comment]
LGTM～
 (1/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (1319.36 s)
 (2/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2012.x86_64.r2.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (1366.04 s)
 (3/3) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native.q35: PASS (1282.87 s)

Acked-by: Peixiu Hou phou@redhat.com

**Review by peixiu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
> @PaulYuuu Yihuang, please review the PR again, thanks.

@CongLi, I just pointed out the configuration problem of cfg, but I don’t know much about the installation of windows.

**Comment by CongLi** [comment]
Thanks @PaulYuuu .
@YongxueHong Please take a look.

**Comment by YongxueHong** [comment]
Thanks all.

**Pull request merged**